test/e2e/certificate_publisher_test.go

```diff
@@ -9,8 +9,8 @@ import (
 
 	operatorv1 "github.com/openshift/api/operator/v1"
 
-	iov1 "github.com/openshift/cluster-ingress-operator/pkg/api/v1"
-	ingresscontroller "github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
+	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
```

Comment:
Maybe import as `operatorcontroller`?  We are inconsistent in how we import this package, and `operatorcontroller` is the least ambiguous of the options in use.

Reply:
Agree, but not sure I agree with another round of CI for it...